jenkins-builds/common-SLES12-2016.10

```diff
@@ -14,4 +14,13 @@ likwid-4.1.2.eb
 HDFView-2.13.0.eb
 ctags-5.8.eb
 lynx-2.8.8rel.2.eb
-tmux-2.3.eb
\ No newline at end of file
+tmux-2.3.eb
+# --- scorep
+zlib-1.2.8-scorep.eb
```

Comment:
> we need to build 'common' and then the system-specific one
This does not make sense: the common stack is compatible with SLES12 based systems and all compilers hence i do not get why you would need to rebuild it for a specific system...